### PR TITLE
BBS-146: Migrate admin

### PR DESCRIPTION
**Pull request by kasperg**
This PR moves the administration interface for the Ting module to the Opensearch provider.

### CODE REVIEW

File: modules/opensearch/opensearch.install [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * Implements hook_install().
+ */
+function opensearch_install() {
+  opensearch_migrate_settings();
+}
+
+/**
+ * Rename old ting variables to opensearch scope.
+ */
+function opensearch_migrate_settings() {
```
Comment (danquah) [inline modules/opensearch/opensearch.install:13]: 👏

File: modules/opensearch/opensearch.module [hunk 2]
```diff
@@ -22,6 +25,135 @@ function opensearch_ding_provider() {
...
+function opensearch_admin_settings($form_state) {
+  $form = array();
+
+  $form['opensearch'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Opensearch service settings'),
+    '#tree' => FALSE,
+  );
+
+  // TODO BBS-SAL: The next couple of settings seems opensearch specific, move
```
Comment (danquah) [inline modules/opensearch/opensearch.module]: This comment seems outdated now :)

**Review by danquah — CHANGES_REQUESTED** [review]
Just that one comment

File: modules/opensearch/opensearch.module (hunk 2 above)
Comment (kasperg) [reply modules/opensearch/opensearch.module]: :+1: I have updated the comment to be focused on ting_agency.

**Pull request merged**